### PR TITLE
Raymond/update types

### PR DESCRIPTION
**Pull request by rjchee**
@postster @syamp

### CODE REVIEW

**Comment by postster** [comment]
LGTM, but I would like @syamp to take a look too.

File: metricchecks/check_impl.go [hunk 1]
```diff
@@ -179,21 +185,21 @@ func (c *checker) InsertMetricValuesFromContext(m *metrics.MetricContext) error
...
 		sname := name + "_string"
 		c.sc.Insert(types.NewConst(0, c.pkg, sname,
-			types.New("string"), exact.MakeString(fmt.Sprintf("%0.2f", metric.Get()))))
+			types.Typ[types.String], exact.MakeString(fmt.Sprintf("%0.2f", metric.Get()))))
 	}
 	for metricName, metric := range m.Counters {
 		name := strings.Replace(metricName, ".", "_", -1) + "_current"
 		c.sc.Insert(types.NewConst(0, c.pkg, name,
-			types.New("float64"), exact.MakeUint64(metric.Get())))
+			types.Typ[types.Float64], exact.MakeUint64(metric.Get())))
```
Comment (rjchee) [inline metricchecks/check_impl.go]: @syamp 
I saw that the old code created a float64 type but appears to compare it to a uint64, so I just mirrored this difference. Should the type be float64 or uint64?

Comment (syamp) [inline metricchecks/check_impl.go]: If it is a counter - you should compare with uint64.

**Comment by syamp** [comment]
lgtm

**Pull request merged**